IngilevichVarvara/cafeMenu.js

```diff
@@ -0,0 +1,250 @@
+//Menu items
+
+//Hamburgers
+//Hamburger types of size
+Hamburger.SIZE_SMALL = {name: "SMALL", price: 50, calories: 20, type: "hamburger"};
```

Comment:
Usually, we use single quotes in JavaScript when working with objects and strings: `SMALL`, `hamburger`.
Please, change it for the rest of the file and use single quotes in the future.

Exception: we use double quotes in some cases by convention, for example when we work with JSX objects in React:

```
<Component name="Sara" />;
```

You can find some tools for your favorite text editor. I recommend using [Prettier](https://prettier.io). It's easy to install as extension in VSCode, for example.